app/views/articles/tags/_sidebar.html.erb

```diff
@@ -34,6 +34,22 @@
         </div>
       </div>
     <% end %>
+    <% if @moderators.present? %>
+      <div class="widget">
+        <header>
+          <h4>tag moderators</h4>
+        </header>
+        <div class="widget-body">
+          <% @moderators.each do |user| %>
+            <div class="widget-user-pic">
+              <a href="/<%= user.username %>">
```

Comment:
I would also add the username as the href title, so that if I hover I know who the user is without having to click, what do you think?

Reply:
Sounds great!  I'll send the change up now.